spec/collectionspace/converter/publicart/movement_spec.rb

```diff
@@ -28,26 +28,38 @@
 
     context 'fields overridden by publicart' do
       context 'authority/vocab fields' do
```

Comment:
Use test pattern shown [here](https://github.com/collectionspace/cspace-converter/wiki/CODE:-Methods-and-patterns-for-Rspec-tests#non-repeatable-fields-with-multiple-possible-source-authorities).

Once all relevant rows/authority source assignments were being tested the spec failed because row 11 of CSV was not populated